docs/spark-streaming-eventhubs-integration.md

```diff
@@ -2,6 +2,19 @@
 The Spark Streaming integration for Azure Event Hubs provides simple parallelism, 1:1 correspondence between Event Hubs 
 partitions and Spark partitions, and access to sequence numbers and metadata.
 
+## Table of Contents
+* [Linking](https://github.com/Azure/azure-event-hubs-spark/blob/master/docs/spark-streaming-eventhubs-integration.md#linking)
```

Comment:
this should be just be `[Linking](#linking)`. Same goes for all the other links. Can you update them?